activation/atxdb.go

```diff
@@ -228,8 +240,8 @@ func (db *ActivationDb) SyntacticallyValidateAtx(atx *types.ActivationTx) error
 		if !bytes.Equal(atx.Commitment.MerkleRoot, atx.CommitmentMerkleRoot) {
 			return errors.New("commitment merkle root included in challenge is not equal to the merkle root included in the proof")
 		}
-
-		if err := db.nipstValidator.VerifyPost(atx.Commitment, atx.Nipst.Space); err != nil {
+		id := signing.NewPublicKey(util.Hex2Bytes(atx.NodeId.Key))
```

Comment:
you need to use `pub` and not the ID from `NodeId`, otherwise there is no sig validation for first ATXs